share/jupyter/kernels/xcpp/kernel.json.in

```diff
@@ -1,9 +1,16 @@
 {
   "display_name": "cpp 14 (xcpp)",
+  "env": {
+      "PATH":"@XEUS_CPP_PATH@",
+      "LD_LIBRARY_PATH":"@XEUS_CPP_LD_LIBRARY_PATH@"
```

Comment:
I merged, but I wonder why we need to set these up exactly, if it wasn't required before?